tests/javascripts/copyToClipboard.test.js

```diff
@@ -346,11 +350,11 @@ describe('copy to clipboard', () => {
 
         test("the copied selection (range) should start after the prefix of the id", () => {
 
-          // that selection (a range) should have a startOffset past the first two nodes:
-          // index 0: text node containing the whitespace before the prefix
-          // index 1: the prefix node
+          // that selection (a range) should have a startOffset of 1:
+          // index 0: the prefix node
```

Comment:
```suggestion
          // index 0: the hidden prefix node e.g. "ID: "
```

Reply:
Easier to understand with some real examples I think:
```suggestion
          // index 0: the visually hidden prefix node, for example "Template ID: " or "API key: "
```

Reply:
Yes, that's much more useful.